### ISSUE TITLE
renderType=har no work

### ISSUE THREAD

**Issue by xsbchen** [prerender/prerender#481]
```js
// server.js
const prerender = require('prerender');
const server = prerender();
server.start();
```

1. run `node server.js`
2. test with `curl http://localhost:3000/render?renderType=har&url=https://www.example.com/`
3. result:
```
NodeError: The "string" argument must be one of type string, buffer, or arrayBuffer. Received type object
    at Function.byteLength (buffer.js:510:11)
    at Object.server._send (d:\prerender\node_modules\prerender\lib\server.js:378:44)
    at firePluginEvent.then.catch (d:\prerender\node_modules\prerender\lib\server.js:292:9)
    at <anonymous>
```

On Chrome/65.0.3312.0, prerender/5.2.1

**Comment by thoop** [comment]
This is working for me in Chrome 64, but with that error I suspect this could be an issue with a newer version of Node instead of Chrome 64 vs 65. Which version of Node are you using?

**Comment by thoop** [comment]
Yeah I think that's the issue. I'll put in a fix and push it out soon.

**Comment by thoop** [comment]
This should be fixed now in v5.2.2. Thanks for reporting it!

**Comment by xsbchen** [comment]
@thoop ths
